main/mafia.hs

```diff
@@ -500,14 +502,39 @@ ghciArgs extraIncludes paths = do
 
   extras <- concat <$> mapM reifyInclude extraIncludes
 
-  let dirs = ["src", "test", "gen", "dist/build/autogen"] <> extras
-  includes  <- catMaybes <$> mapM ensureDirectory dirs
+  let
+    dirs =
+      ["src", "test", "gen", "dist/build/autogen"] <> extras
```

Comment:
Yeah I have the same issue with `hsc2hs`, it would be `dist/build` recursively right?